log-base/src/Log/Logger.hs

```diff
@@ -32,6 +32,7 @@ data LoggerEnv = LoggerEnv
   , leDomain    :: ![T.Text] -- ^ Current application domain.
   , leData      :: ![A.Pair] -- ^ Additional data to be merged with the log
                              -- message\'s data.
+  , leMaxLogLevel :: LogLevel -- ^ The maximum log level to be logged.
```

Comment:
To make things very clear that it's the maximum (not minimum) allowed level, we should probably add a note to where `LogLevel` is defined and explicitly mention that it derives `Ord`. And the order in which the constructors are defined then implies which one is actually of "the highest value".